app/src/main/java/eu/epitech/reyditech/Reyditech.kt

```diff
@@ -14,16 +33,121 @@ internal const val USER_AGENT: String = "android:eu.epitech.reyditech:v1.0.0 (by
  * The root component of Reyditech.
  * Handles navigation between screens and "global" state.
  */
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
 @Composable
+@Preview
 internal fun Reyditech() {
     val navController = rememberNavController()
+    var searchparam by remember { mutableStateOf(TextFieldValue("")) }
+    Theme {
+        Scaffold(topBar = {
+            TopAppBar(backgroundColor = MaterialTheme.colors.primaryVariant,
+                modifier = Modifier.height(63.dp),
+                title = {
+                    Row(
+                        modifier = Modifier.fillMaxWidth(),
+                        verticalAlignment = Alignment.CenterVertically
+                    ) {
+                        Icon(
+                            painter = painterResource(id = R.drawable.dark_logo),
+                            contentDescription = "Logo",
+                            tint = Color.Unspecified
+                        )
+                        Text(
+                            text = "Reyditech"
 
-    NavHost(navController = navController, startDestination = "login") {
-        composable("login") {
-            LoginScreen(onLoginFinished = { navController.navigate("main") })
-        }
-        composable("main") {
-            MainScreen(onReLogin = { navController.navigate("login") })
-        }
+                        )
+                        OutlinedTextField(
+                            modifier = Modifier
+                                .background(Transparent)
+                                .padding(bottom = 2.dp),
+                            textStyle = androidx.compose.ui.text.TextStyle(fontSize = 15.sp),
+                            value = searchparam,
+                            onValueChange = {
+                                println(searchparam)
+                                searchparam = it
+                            },
+                            label = { Text(text = "search") },
```

Comment:
Le text est pas traduit
Pro tip: met ton cursor sur la string, fait ALT+ENTER, puis clique sur extract to resource